Sources/CodeEditTextView/Utils/CEUndoManager.swift

```diff
@@ -164,18 +172,18 @@ public class CEUndoManager {
     // MARK: - Grouping
 
     /// Groups all incoming mutations.
-    public func beginGrouping() {
```

Comment:
Does it make sense for these to be private? They are never used publicly.

Reply:
If they are never used publicly then making them private may make sense. @thecoolwinter did you have any plans to use these anywhere else?